demo/site/src/pages/news/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { PreviewData } from "@comet/cms-site";
+import { DamImageBlock } from "@src/blocks/DamImageBlock";
+import { defaultLanguage, domain } from "@src/config";
+import createGraphQLClient from "@src/util/createGraphQLClient";
+import { gql } from "graphql-request";
+import { GetServerSideProps, InferGetServerSidePropsType } from "next";
+import Link from "next/link";
+import { ParsedUrlQuery } from "querystring";
+import { FormattedDate } from "react-intl";
+import styled from "styled-components";
+
+import { GQLNewsIndexPageQuery, GQLNewsIndexPageQueryVariables } from "./index.generated";
+
+export default function NewsIndexPage({ newsList }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+    return (
+        <div>
+            <h1>News</h1>
+            <CardList>
+                {newsList.nodes.map((news) => (
+                    <Link key={news.id} href={`/news/${news.slug}`} passHref>
+                        <Card>
+                            <DamImageBlock data={news.image} aspectRatio="4x3" />
+                            <h2>{news.title}</h2>
+                            <p>
+                                <FormattedDate value={news.createdAt} />
+                            </p>
+                        </Card>
+                    </Link>
+                ))}
+            </CardList>
+        </div>
+    );
+}
+
+export const getServerSideProps: GetServerSideProps<GQLNewsIndexPageQuery, ParsedUrlQuery, PreviewData> = async (context) => {
+    const client = createGraphQLClient(context.previewData);
+    const locale = context.locale ?? defaultLanguage;
+    const scope = { domain, language: locale };
+
+    const data = await client.request<GQLNewsIndexPageQuery, GQLNewsIndexPageQueryVariables>(
+        gql`
+            query NewsIndexPage($scope: NewsContentScopeInput!, $sort: [NewsSort!]!) {
+                newsList(scope: $scope, sort: $sort) {
+                    nodes {
+                        id
+                        title
+                        slug
+                        image
+                        createdAt
+                    }
+                }
+            }
+        `,
+        { scope, sort: [{ field: "createdAt", direction: "DESC" }] },
```

Comment:
no need to pass hardcoded as variable, simply hardcode in the query itself

Reply:
I've once read that hardcoding variables isn't a best practice due to decreased server caching:

> If two otherwise identical queries have different hardcoded argument values, they're considered entirely different operations by your GraphQL server's cache. The cache enables your server to skip parsing and validating operations that it's encountered before, improving performance.

https://www.apollographql.com/docs/react/data/operation-best-practices/#disadvantages-of-hardcoded-graphql-arguments

But of course this is no issue here, I'm just doing it out of habit. Should I change it?

Reply:
Interesting. Now we only need a server cache. Do whatever you think is best :)

Reply:
Apollo Server includes some sort of cache by default. But don't ask me what it's used for 😁